app/logic/spreadsheet.py

```diff
@@ -143,9 +145,9 @@ def termParticipation(termDescription):
 
     return programParticipationDict
 
-def removeNullParticipants(bla):
+def removeNullParticipants(participant_list):
```

Comment:
use camelCase to stay consistent

Reply:
I refactored the entire spreadsheets.py to be more consistent, since on the page, both camelCase and underscores were used.